packages/sui-mono/bin/sui-mono-release.js

```diff
@@ -142,9 +147,61 @@ const checkIsMasterBranchActive = async ({status, cwd}) => {
   }
 }
 
+const execute = async (cmd, full) => {
```

Comment:
Where are you using this function?

Reply:
https://github.com/SUI-Components/sui/pull/907/files#diff-07d9dd6c75c76a2e1d0a25dc5fd3e0cbR170

Reply:
L 170

Reply:
in line 170